gizmo.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import numpydoc
+import pandas as pd
+
+def hello(name, country = 'Finland'):
+    print('Hello ', name, ', how are things in ', country, '?', sep = '')
```

Comment:
alternatively, you can use f-strings here: `print((f'Hello, {name}, how are things in {country}?')`